test/parallel/test-buffer.js

```diff
@@ -678,6 +678,14 @@ for (var i = 0; i < 256; i++) {
   assert.equal(hexb2[i], hexb[i]);
 }
 
+console.log('Create buffer from odd-length hex string should fail');
+assert.throws(function() {
+  var hexb3 = new Buffer('8', 'hex');
+}, function(err) {
+  return err instanceof TypeError &&
+      err.message == "The hex string must have an even number of characters";
```

Comment:
Can you use `===` here. The linter will complain about the double quotes as well.